### DISCUSSION TITLE
Rolladen Stop funktioniert nicht

### DISCUSSION THREAD

**Discussion by mariushun** [brutella/hkknx-public#207] [Q&A] [answered]
Hi, 

ich habe einen Hager Domovea Basic TJA670 KNX Server und würde gerne meine Rolläden über HKKNX steuern. Die Positionsangabe in % funktioniert einwandfrei, Auf/Ab theoretisch auch. Wenn der Rolladen ganz oben ist und ich tippe nur auf das Symbol beginnt er runterzufahren. Wenn ich nun nochmal auf das Symbol drücke um zu stoppen, stoppt der Rolladen, aber fährt direkt wieder ganz hoch. 

Meine Stopadresse ist in Domovea vom DPT 1.007. Ist das vielleicht das Problem? Ich bin leider kein Profi in KNX, aber bisher hat alles super funktioniert.

Danke vorab für eure Ideen.

**Comment by chef76** [comment, marked as answer]
In Homekit gibt es kein Stop für Rolladen. Du musst direkt eine Position in % anfahren (eigene Gruppenadresse).

**Comment by brutella** [comment]
> Wenn der Rolladen ganz oben ist und ich tippe nur auf das Symbol beginnt er runterzufahren. Wenn ich nun nochmal auf das Symbol drücke um zu stoppen, stoppt der Rolladen, aber fährt direkt wieder ganz hoch.

So verhält sich das in der Apple Home-App. Das hat nichts mit einer falschen Konfiguration oder hkknx zu tun. In der Home-App ist es so implementiert, dass wenn man während der Fahrt eines Rollladens auf das Symbol klickt, das die Fahrt umgekehrt wird. Dass kann man leider nicht ändern.

HomeKit unterstützt eine Stopp-Funktion für Rollläden. Diese Funktion wird aber von der Apple Home-App nicht verwendet.

**Comment by chef76** [comment]
> HomeKit unterstützt eine Stopp-Funktion für Rollläden. Diese Funktion wird aber von der Apple Home-App nicht verwendet.

... oder so. Ist schon etwas her, dass wir das diskutiert haben. :-) Ich hatte nur noch im Kopf, dass es in Home nicht geht.

**Comment by mariushun** [comment]
Danke euch für die schnellen Antworten. :-) Schade trotzdem.

**Comment by Damageplan1980** [comment]
Hi zusammen,

ich bin auch gerade am HKKNX testen und bin ebenfalls über das Thema Jalousiesteuerung gestolpert.
Wenn man die Jalousie nicht stoppen kann in der HomeKit App, wozu ist dann in HKKNX die "Jalousie stoppen" Eigenschaft in der Konfiguration eine Jalousie vorhanden?
Wo und wie kann ich das nutzen?

Danke für eure Hilfe und viele Grüße
Matthias

**Reply by brutella** [reply]
> Wenn man die Jalousie nicht stoppen kann in der HomeKit App, wozu ist dann in HKKNX die "Jalousie stoppen" Eigenschaft in der Konfiguration eine Jalousie vorhanden? 

HomeKit unterstützt eine Stopp-Funktion von Jalousien, nur die Home-App verwendet es nicht.

> Wo und wie kann ich das nutzen?

Die [Home+](https://itunes.apple.com/us/app/home-smart-home-automation/id995994352) zeigt dir beispielsweise die Halteposition-Eigenschaft in der Detailansicht für ein Gerät an.

---

Wie würdest du dir das Verhalten eines Rollladens in HomeKit wünschen?
Wenn man während dem Öffnen/Schließen tippt, dass der Rollladen dann stehen bleibt?
Was soll dann beim nochmaligem tippen geschehen – die Fahrt umkehren?

**Reply by dreamdatcrap** [reply]
Hi zusammen,

jetzt muss ich mich auch mal einschalten. 

Zuerst, Matthias, vielen Dank für all deine Mühen!

So wie ich das lese habe ich genau das selbe Problem. 

Wäre super, dass der Rollladen stoppt, sobald man einmal auf die Taste tippt. Wird dann die hoch oder runter Taste gehalten, wird die jeweilige Richtung angefahren. Wird nur getippt, werden die Lamellen bewegt

gruß

**Reply by brutella** [reply]
@dreamdatcrap Wenn du gerne dieses Verhalten in der Apple Home-App haben möchtest, muss ich dich enttäuschen. Das wird so nicht funktionieren. 

Was jedoch möglich ist, dass ich in [Home+](https://itunes.apple.com/us/app/home-smart-home-automation/id995994352) eine Funktion einbaue, die den Rollladen zum stehen bringt, wenn während der Fahrt auf das Rollladen-Feld getippt wird.

[a comment by an automated account is not shown]
